### PR TITLE
max. billing unit lag help message

### PR DESCRIPTION
**Pull request by EKorobov**
implemented link opening with `require('electron').shell.openExternal()`, but not sure is this good way to do this, it is working on Linux, but i'm not sure about Windows and Mac

### CODE REVIEW

**Comment by drew2a** [comment]
I'll check it on mac.

File: src/agent/offerings/createOffering/index.tsx [hunk 1]
```diff
@@ -381,6 +381,11 @@ class CreateOffering extends React.Component<IProps, any>{
         }
     }
 
+    openExternalLink(url: string) {
+        // TODO: mpt sure is this ok on all systems
+        require('electron').shell.openExternal(url);
```
Comment (drew2a) [inline src/agent/offerings/createOffering/index.tsx]: is it closely correlates to https://privatix.atlassian.net/browse/BV-1182?

Comment (EKorobov) [reply src/agent/offerings/createOffering/index.tsx]: i'm not sure that these tasks are connected

Comment (EKorobov) [reply src/agent/offerings/createOffering/index.tsx]: but i'm not sure though

**Comment by EKorobov** [comment]
To avoid splitting vocabulary constants, I've add `props.children` usage to 'ExternalLink` component. It allows us to set link text via children (`<ExternalLink href={'http://qwe.asd/'}>Link Text</ExternalLink>`) insted of using attributes (`<ExternalLink href={'http://asd.qwe'} text={'My Link'} />`). As a result, `ExternalLink` can be used inside `Trans` component without any i18n problems.

**Pull request merged**